larray/core/array.py

```diff
@@ -5644,9 +5644,7 @@ def opmethod(self, other) -> 'Array':
             # TODO: ndarray should probably be converted to larrays because that would harmonize broadcasting rules, but
             # it makes some tests fail for some reason.
             if not isinstance(other, (Array, np.ndarray)) and not np.isscalar(other):
-                if other is None:
-                    other = full(self.axes, fill_value=None)
-                elif isinstance(other, (list, Axis)):
+                if isinstance(other, (list, Axis)) or other is None:
```

Comment:
Well, in fact, I think the best fix is to move the "is None" check in the line above, so that other is passed as is to the numpy binop

```python
if other is not None and not isinstance(other, (Array, np.ndarray)) and not np.isscalar(other):
```

and when looking more at this code, I think it's weirdly written (I know, I was the one to write it!), the two different levels of "if" clause have no point. The following code should be equivalent (not tested though!) and clearer IMO:

```python
            if isinstance(other, (list, Axis)):
                other = asarray(other)
            elif other is not None and not np.isscalar(other) and not isinstance(other, (Array, np.ndarray)):
                # support for inspect.signature
                return False
```